airflow/providers/elasticsearch/CHANGELOG.rst

```diff
@@ -27,6 +27,18 @@
 Changelog
 ---------
 
+5.1.0
+.....
+
+Misc
+~~~~
```

Comment:
Please remove above including the version.
It is not known yet which version is going to be next.. it could be 5.0.1, 5.1.0, 6.0.0
we know the release version only in run time.

The change we need here is just the note after the changelog title.. during release the release manager set it to the proper classification

Reply:
Sure. Removed.